File: prog02.c
#include <stdio.h>

int main() {
    
    int idade;

    printf("Digite sua idade: ");
    scanf("%d", &idade);

    if (idade < 18) {
        if (idade < 6) {
            printf("Nao paga passagem (crianca).\n");
        } else {
            printf("Paga meia passagem (estudante).\n");
        }
    } else {
        if (idade < 60) {
            printf("Paga passagem inteira.\n");
        } else {
            printf("Nao paga passagem (idoso).\n");
        }
    }

    return 0;
}

File: prog02.py
idade = int(input("Digite sua idade: "))

if idade < 18:
    if idade < 6:
        print("Nao paga passagem (crianca).")
    else:
        print("Paga meia passagem (estudante).")
else:
    if idade < 60:
        print("Paga passagem inteira.")
    else:
        print("Nao paga passagem (idoso).")